examples/triton/tensorflow/helpers.py

```diff
@@ -0,0 +1,409 @@
+from __future__ import annotations
+
+import os
+import json
+import time
+import typing as t
+from pathlib import Path
+
+import fs
+import attr
+import torch
+import fs.errors
+
+from bentoml._internal.utils import LazyLoader
```

Comment:
I think when 3.12 is officially out and modules are lazy loaded by default, it would be nice.

I think we can remove all of the internal utils here.